packages/interceptors/README.md

```diff
@@ -28,10 +28,10 @@ yarn add @thi.ng/interceptors
 ## Usage examples
 
 ```ts
-import * as interceptors from "@thi.ng/interceptors";
+importas interceptors from "@thi.ng/interceptors";
```

Comment:
Was this edit intentional?   Looks like a typo.

Reply:
ah, that's a typo... sorry

Reply:
Haha thanks.  The way JavaScript moves these days I had to double check.  Results were  mostly in Spanish.

Reply:
haha